server/utils/validation/workValidation.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require('joi');
+
+const workValidation = Joi.object({
+  title: Joi.string().max(50).required(),
+  content: Joi.string().required(),
+  // image: Joi.string().dataUri().required(),
```

Comment:
```suggestion
   image: object()
    .test('type', 'Should be an image png or jpeg', (value) => {
      return ['image/png', 'image/jpeg'].includes(value.type);
    })
    .required(),
```
Try this might help